src/jpegParser.js

```diff
@@ -228,8 +241,13 @@
     var processMetaData = function(error, metaData) {
       var thumbnailMetaData = metaData && metaData.thumbnailMetaData;
       var thumbnailBlob = metaData && metaData.thumbnailBlob;
+      var imageSize = metaData && {
+        width: metaData.width,
+        height: metaData.height,
+        progressive: metaData.progressive
+      };
       metaData = metaData && metaData.Exif;
-      callback(error, metaData, thumbnailMetaData, thumbnailBlob);
+      callback(error, metaData, thumbnailMetaData, thumbnailBlob, imageSize);
```

Comment:
Thi list of arguments is getting too long. Should we make an object with the data? callback(error, data) 
data {
exifMetadata : {}
 thumbnailMetaData : {}
thumbnailBlob: 
imageSize {}
}